### ISSUE TITLE
Add hotkeys for 'Ok' and 'revert' of detection cropping tool

### ISSUE THREAD

**Issue by sanecito** [openstreetcam/openstreetcam.org#186]
OSC does an excellent job making it quick to validate signs using the Ctrl+ arrow keys. It would be useful if this also extended to the 'ok' and 'revert' buttons on the detection crop tool. I suggest space for 'ok' and 'z' for revert (given z is semi-synonymous with ctrl+z).

**Comment by alexandrui-telenav** [comment]
good point, not entirely sure what you mean by 'ok' though: is it after you select the bounding box and sign, to 'save'?

**Comment by sanecito** [comment]
Yeah, the green checkmark icon.
